test/test_encryption.py

```diff
@@ -2202,6 +2202,93 @@ def test_05_roundtrip_encrypted_unindexed(self):
         self.assertEqual(decrypted, val)
 
 
+# https://github.com/mongodb/specifications/blob/072601/source/client-side-encryption/tests/README.rst#rewrap
+class TestRewrapWithSeparateClientEncryption(EncryptionIntegrationTest):
+
+    MASTER_KEYS: Mapping[str, Mapping[str, Any]] = {
+        "aws": {
+            "region": "us-east-1",
+            "key": "arn:aws:kms:us-east-1:579766882180:key/89fcc2c4-08b0-4bd9-9f25-e30687b580d0",
+        },
+        "azure": {
+            "keyVaultEndpoint": "key-vault-csfle.vault.azure.net",
+            "keyName": "key-name-csfle",
+        },
+        "gcp": {
+            "projectId": "devprod-drivers",
+            "location": "global",
+            "keyRing": "key-ring-csfle",
+            "keyName": "key-name-csfle",
+        },
+        "kmip": {},
+        "local": {},
+    }
+
+    KMS_TLS_OPTS = {
```

Comment:
I think the tlsCAFile is only valid/needed for kmip. Can we use the existing KMS_TLS_OPTS global instead of adding a new self.KMS_TLS_OPTS?

Reply:
Done